bee-network/bee-autopeering/Cargo.toml

```diff
@@ -32,19 +32,19 @@ num-derive = { version = "0.3.3", default-features = false  }
 num-traits = { version = "0.2.14" , default-features = false }
 priority-queue = { version = "1.2.0" , default-features = false }
 prost = { version = "0.8", default-features = false, features = [ "std" ] }
-rand = { version = "0.8.4" , default-features = false, features = [ "std", "std_rng" ] }
+rand = { version = "0.8.5" , default-features = false, features = [ "std", "std_rng" ] }
 ring = { version = "0.16.20" , default-features = false }
-rocksdb = { version = "0.17.0", default-features = false, optional = true }
+rocksdb = { version = "0.18.0", default-features = false, optional = true }
 serde = { version = "1.0", default-features = false , features = [ "derive" ] }
```

Comment:
`serde` and `tokio` are sometimes pinned to a specific `patch` version and sometimes we omit the `patch`. Maybe we should unify that?

Reply:
Will do